_posts/2018-01-07-building-serverless-radiator.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: Building a serverless radiator for fun and benefit with Clojure, Python and AWS 
+author: hjhamala
+excerpt: Monitoring software development pipeline statuses, environment alarms and metrics are an essential part of a modern software development process in a cloud environment. In this post I show a simple way to monitor different accounts with a serverless radiator. 
+tags:
+- AWS
+- Clojure
+- Serverless 
+---
+
+***There exists many commercial products for monitoring development pipelines and deployments. What could be more fun than make your own?***
+
+**Modern** software development environment is most likely composed of multiple pipelines. There can be pipelines for build artifacts, infrastructure building etc. It is vital to notice fast when the builds fail. Monitoring only the pipelines is not necessary enough. Also deployments should be monitored for noticing potential problems in performance etc. 
```

Comment:
Ehkä "notice build failures as soon as possible".

"Only monitoring…"

"Also, deployments should be monitored to detect potential…"